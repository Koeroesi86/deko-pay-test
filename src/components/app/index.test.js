import React from 'react';
import { shallow } from 'enzyme';
import App from './index';

describe('The App component', () => {
  test('renders', () => {
    const result = shallow(<App />);
    expect(result).toMatchSnapshot();
  });
});
