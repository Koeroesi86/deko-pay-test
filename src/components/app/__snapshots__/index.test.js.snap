// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App component renders 1`] = `
<div
  className="App"
>
  <Merchants
    selected=""
    setSelected={[Function]}
  />
  <Transactions
    merchant=""
  />
</div>
`;
